gatherers/censys.py

```diff
@@ -127,45 +128,61 @@ def gather(suffixes, options, extra={}):
 #     OR sans LIKE "%.gov")
 #   OR (common_names LIKE "%.fed.us"
 #     OR sans LIKE "%.fed.us");
+def query_for(suffixes: List[str]) -> str:
 
-def query_for(suffixes):
+    select = "\n".join([
+        "    parsed.subject.common_name,",
+        "    parsed.extensions.subject_alt_name.dns_names",
+    ])
 
-    select = """
-    parsed.subject.common_name,
-    parsed.extensions.subject_alt_name.dns_names
-    """
-
-    from_clause = """
-    `censys-io.certificates_public.certificates`,
-    UNNEST(parsed.subject.common_name) AS common_names,
-    UNNEST(parsed.extensions.subject_alt_name.dns_names) AS sans
-    """
+    from_clause = "\n".join([
+        "    `censys-io.certificates_public.certificates`,",
+        "    UNNEST(parsed.subject.common_name) AS common_names,",
+        "    UNNEST(parsed.extensions.subject_alt_name.dns_names) AS sans",
+    ])
```

Comment:
I don't have strong feelings either way, but what makes `"\n".join([ ... ])` better than using `"""..."""` syntax for long multi-line strings?

Reply:
It's much easier to see whitespace differences. With the multi-line strings you have to conform to the indentation of the surrounding block.
Using the join approach makes it very clear when you want to add space on purpose versus when you're just doing that to satisfy the parser.